src/mpyl/steps/deploy/k8s/resources/dagster.py

```diff
@@ -0,0 +1,52 @@
+"""
+This module contains the Dagster user-code-deployment values conversion
+"""
+from dataclasses import dataclass
+
+from mpyl.project import Project, get_env_variables
+from mpyl.steps.models import RunProperties
+from mpyl.utilities.docker import DockerConfig
+
+
+@dataclass(frozen=True)
+class Constants:
+    DAGSTER_NAMESPACE = "dagster"
+
+    DAGSTER_WORKSPACE_CONFIGMAP = "dagster-workspace-yaml"
+    DAGSTER_WORKSPACE_FILE = "workspace.yaml"
+    DAGSTER_DAEMON = "dagster-daemon"
+    DAGSTER_DAGIT = "dagster-dagit"
+
+    HELM_CHART_REPO = "https://dagster-io.github.io/helm"
+    CHART_NAME = "dagster/dagster-user-deployments"
```

Comment:
Are any of these values specific to our setup?

Reply:
i think so. (except HELM_CHART_REPO and CHART_NAME) -> as a follow up i would like to also specific the `DagsterDeploy`-step as a `DagsterUserCodeDeploy` step, but for the sake of our transition from MPL to MPyL, we have this intermediate state.

Do you think mpyl-config, or project.yml would be a place for the other constants?

Reply:
a dagster section in the mpyl config seems like a good choice